### PR TITLE
Added WatchOS 2 support

### PR DESCRIPTION
**Pull request by jaernouts**

### CODE REVIEW

File: Reliant.podspec [hunk 1]
```diff
@@ -13,11 +13,15 @@ Pod::Spec.new do |s|
...
 
+  s.watchos.deployment_target = '2.0'
+  s.watchos.exclude_files = 'Reliant/Classes/ContextLocator/OSX/*', 'Reliant/Classes/ContextLocator/iOS/*', 'Reliant/Classes/Categories/UIKit/*'
+
   s.osx.deployment_target = '10.8'
   s.osx.exclude_files = 'Reliant/Classes/Categories/UIKit/*', 'Reliant/Classes/ContextLocator/iOS/*'
 
   s.frameworks = 'Foundation'
   s.ios.frameworks = 'UIKit'
+  s.watchos.frameworks = 'UIKit'
```
Comment (mikeseghers) [inline Reliant.podspec]: UIKit as watchos framework seems weird?

**Comment by mikeseghers** [comment]
Thanks a lot Jens! I will try to make some time to check why travis is failing.

File: Reliant.podspec (hunk 1 above)
Comment (jaernouts) [inline Reliant.podspec]: Correct, will remove that!

**Comment by mikeseghers** [comment]
Hi Jens,

could you pull jaernouts-master, so we can keep everything tested in your fork?

**Comment by jaernouts** [comment]
Hi Mike, it should be done now!

**Comment by mikeseghers** [comment]
HI Jens,
I've been running the tests locally, and they don't seem to fail, but travis still experiences trouble, so I'll need some more time to investigate. I'm on it though.

**Comment by mikeseghers** [comment]
Hi Jens, can you pull again?

**Comment by jaernouts** [comment]
Done!

**Comment by mikeseghers** [comment]
Great succes! I will merge and bump the pod version to 2.3.0.

**Comment by mikeseghers** [comment]
Hi Jens, just to let you know that Reliant 2.3.0 is available on CocoaPods

**Comment by jaernouts** [comment]
Super, thanks!

**Pull request merged**